src/VisualStudio/Core/Def/Implementation/FindUsages/VisualStudioFindSymbolMonikerUsagesService.cs

```diff
@@ -43,43 +43,37 @@ internal partial class VisualStudioFindSymbolMonikerUsagesService : AbstractFind
         }
 
         public override async IAsyncEnumerable<ExternalReferenceItem> FindReferencesByMoniker(
-            DefinitionItem definition, ImmutableArray<SymbolMoniker> monikers,
-            IStreamingProgressTracker progress, [EnumeratorCancellation] CancellationToken cancellationToken)
+            DefinitionItem definition, ImmutableArray<SymbolMoniker> monikers, [EnumeratorCancellation] CancellationToken cancellationToken)
         {
             if (_codeIndexProvider == null)
                 yield break;
 
+            // Only grab the first 500 results.  This keeps server load lower and is acceptable for //build demo purposes.
+            const int PageCount = 5;
```

Comment:
Did this mean to come in this PR or something else?

Reply:
yes.  i pulled this in from my main richnav PR (which i'm still trying to get in).